src/components/Header/Header.css

```diff
@@ -1,7 +1,31 @@
 .header {
+    margin-top: 5.5vh;
+    margin-left: 2vw;
+    margin-right: 3vw;
+    margin-bottom: 5.5vh;
     position: fixed;
-    width: 100vw;
+    width: 95vw;
+    display: flex;
+    flex-direction: row;
+    gap: 10px;
+    align-items: center;
 }
 
 /* TODO: Add styling for your Header component
-here. */
\ No newline at end of file
+here. */
+#profileLink {
```

Comment:
generally it's better to select using `className`s instead of `id`s unless absolutely necessary for css